src/DataTable/sorting.ts

```diff
@@ -0,0 +1,38 @@
+// The directions in which a column can be sorted
+// Note: if a table is sorted, one column _must_ have a direction other than
+// NONE
+export type SortDirection = 'ASC' | 'DESC' | 'NONE'
+
+// A mapping of names to SortDirection
+export const SortDirection: {
```

Comment:
I think when the same identifier is used an `enum` can only merge with another `enum`, unfortunately. I would love to use the same name as no others ones really stuck, let me know if you have any suggestions (name or otherwise!)

Reply:
No ideas 🤷‍♂️ Let's just keep it as a `const`